### DISCUSSION TITLE
0.71.10-0 release is now available

### DISCUSSION THREAD

**Discussion by douglowder** [react-native-tvos/react-native-tvos#516] [Announcements]
The release of React Native TV for 0.71!

Notes:

- Contains all the commits from the previous stable release (0.69.8-2).
- Contains all the commits from React Native core 0.71-stable branch.
- *Hermes running on Apple TV!* The template has been modified to enable this automatically. Please let us know on Discord (or by filing an issue) if there are problems. As Xcode 14 has deprecated bitcode, bitcode generation has been removed from Hermes artifacts, greatly reducing their size.
- In 0.71, the React Native Android prebuilt archives are published to Maven instead of being included in the NPM. We are following the same model, except that the Maven artifacts will be in group `io.github.react-native-tvos` instead of `com.facebook.react`. The `react-native-gradle-plugin` has been upgraded so that the Android dependencies will be detected correctly during build.
- The TV-specific Typescript types, formerly in `tvos-types.d.ts`, are now in `./types/public` and are automatically imported along with the other React Native types when using Typescript.

Known Issues:

- The React Native CLI (`@react-native-community/cli`) should still be used to create new projects with this repo. However, `npx react-native run-ios` will no longer run Apple TV targets. A fix for this is in review at https://github.com/react-native-community/cli/pull/1929 . To run Apple TV (and Android TV) targets from the command line, it is now possible to use the Expo CLI, using the following steps:
  - In your app, install the required Expo modules: `yarn add expo`
  - Add a file `react-native.config.js` at the top level of your app directory, with [these contents](https://github.com/byCedric/custom-prebuild-example/blob/main/app/react-native.config.js).
  - Then an Apple TV target can be run: `npx expo run:ios --scheme MyApp-tvOS --device "Apple TV"`
  - To run Android TV: `npx expo run:android`
  - See [this document](https://docs.expo.dev/bare/using-expo-cli/) for more details on Expo CLI functionality (note that many of these features require that Expo SDK modules be built into your app, which is not yet supported on Apple TV).
  - If desired, you can create a new project using the [react-native-template-typescript-tv](https://github.com/react-native-tvos/react-native-template-typescript-tv) template. This template already has the modifications needed to use the Expo CLI as described above

<hr /><em>This discussion was created from the release <a href='https://github.com/react-native-tvos/react-native-tvos/releases/tag/v0.71.10-0'>0.71.10-0</a>.</em>

**Comment by JS-GitRepo** [comment]
I just want to say that I couldn't get onFocus and onBlur to work on Android TV with React Native...

and react-native-tvos was literally a drop in, plug-n-play solution to the problem! Simply deleted node_modules, replaced "react-native" dependency with "react-native": "npm:react-native-tvos@0.71.10-0" and npm install. Also followed instructions in the release notes to create react-native.config.js. 

Ran expo start --dev-client and everything just works! Didn't even need to rebuild with EAS build. I'm writing this comment because this was the most seamless fix to such an annoying problem, thank you to everyone involved in this project!
